frontend/apps/reader/modules/readerrolling.lua

```diff
@@ -868,21 +868,15 @@ function ReaderRolling:onUpdatePos()
         return true
     end
 
---    if Device:isAndroid() then
---        logger.dbg("readerrolling:onUpdatePos() setIgnoreInput=true")
---        Device:setIgnoreInput(true)
---    end
+    Device:setIgnoreInput(true) -- to avoid ANRs on Android, disable taps
 
     -- Calling this now ensures the re-rendering is done by crengine
     -- so updatePos() has good info and can reposition
     -- the previous xpointer accurately:
     self.ui.document:getCurrentPos()
 
---    if Device:isAndroid() then
---        logger.dbg("readerrolling:onUpdatePos() setIgnoreInput=false")
---        Device:setIgnoreInput(false)
---        UIManager:discardEvents(true)
---    end
+    Device:setIgnoreInput(false) -- allow taps from now
```

Comment:
Does setIgnoreInput really only disable taps ?! but would allow swipe/pinch ?!
If yes, it may be an oversight in setIgnoreInput - it feels it should disable all events (including hardware keys/keyboard, power button...)